File: 005_plotting_a_function/solution/plot_z_coord.m
function plot_z_coord(V,F)
%PLOT_Z_COORD  plot the z-coordinate on the input mesh
%
% plot_z_coord(V,F);
%
% Input:
%  V,F  mesh that is to be plotted

f = V(:,3);
tsurf(F,V, 'CData',f);
axis equal;
shading interp;

end

File: 005_plotting_a_function/solution/plot_z_coord.py
import gpytoolbox as gpy, polyscope as ps, numpy as np

def plot_z_coord(V,F):
    """This method plots the z-cordinate on the input mesh V,F
    """

    f = V[:,2]
    ps.init()
    ps_surface = ps.register_surface_mesh("surface", V, F)
    ps_surface.add_scalar_quantity("z coordinate", f, enabled=True)
    ps.show()


